### ISSUE TITLE
how to achieve realtime object detection + tracking + 2d pose + 3d pose estimetion from a carmera

### ISSUE THREAD

**Issue by vicentowang** [fabro66/GAST-Net-3DPoseEstimation#54]
as you demo video demonstrated. https://www.bilibili.com/video/BV18f4y197R7/

**Comment by wtnan2003** [comment]
@vicentowang hi，Have you achieved that?

**Comment by vicentowang** [comment]
yes yolov5+deepsort+darkpose+gast about 30fps with opengl realtime visualization on 3070 GPU



---Original---
From: "Xiao ***@***.***&gt;
Date: Mon, Dec 6, 2021 18:27 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [fabro66/GAST-Net-3DPoseEstimation] how to achieve realtimeobject detection + tracking + 2d pose + 3d pose estimetion from a carmera(Issue #54)




 
@vicentowang hi，Have you achieved that?
 
—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by Ared521** [comment]
> 



> yes yolov5+deepsort+darkpose+gast about 30fps with opengl realtime visualization on 3070 GPU
> […](#)
> ---Original--- From: "Xiao ***@***.***&gt; Date: Mon, Dec 6, 2021 18:27 PM To: ***@***.***&gt;; Cc: ***@***.******@***.***&gt;; Subject: Re: [fabro66/GAST-Net-3DPoseEstimation] how to achieve realtimeobject detection + tracking + 2d pose + 3d pose estimetion from a carmera(Issue #54) @vicentowang hi，Have you achieved that? — You are receiving this because you were mentioned. Reply to this email directly, view it on GitHub, or unsubscribe. Triage notifications on the go with GitHub Mobile for iOS or Android.

Hello, could you please provide your real-time inference code?

**Comment by sk-zhang** [comment]
Hi, could you also please provide me with your real-time inference code？

**Comment by xobeiotozi** [comment]
Hello, could you please provide your real-time inference code?

**Comment by zdf147258** [comment]
Hello, could you please provide your real-time inference code?
